README.md

```diff
@@ -2,34 +2,35 @@
 
 # NuoDB Collector
 
-[![Build Status](https://travis-ci.org/nuodb/nuodb-collector.svg?branch=master)](https://travis-ci.org/nuodb/nuodb-collector)
+[![nuodb](https://circleci.com/gh/nuodb/nuodb-collector.svg?style=svg)](https://circleci.com/gh/nuodb/nuodb-collector)
 
 # Introduction
 
 Most modern application monitoring systems consist of the following 3
 core components:
 
-* Collector             — daemon(s) to gather metrics such as this repository
-* Time Series database  — for storage of real-time, high volume metrics (e.g. InfluxDB, Prometheus, LogStash)
-* Query & Visualization — that enables visual monitoring and root cause analysis (e.g. Grafana)
+- Collector — daemon(s) to gather metrics such as this repository
+- Time Series database — for storage of real-time, high volume metrics (e.g. InfluxDB, Prometheus, LogStash)
+- Query & Visualization — that enables visual monitoring and root cause analysis (e.g. Grafana)
```

Comment:
This doesn't change anything in terms of formatting, right?

Reply:
the linting is done Prettier.